woocommerce/woocommerce-android
# Issue/5705 add order refund card numbers

Closes: #5705 


### Description

Adds card data to the refund screen.

Notice that the PR doesn't handle:
* Payments made with Stripe extension
* Cases when a user never opened an onboarding screen

@AnirudhBhat do you think are we ok to with that or better to wait for the decision and implement it on the spot?

### Testing instructions

* Open different types of orders and their refund screens
* Notice card details

### Images/gif

![image](https://user-images.githubusercontent.com/4923871/152346439-daf20ebc-4156-4fc2-b1ae-b99a53bf2401.png)
